src/tagging/components/taggingWithButtons.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Row, ButtonGroup, Button, Col } from 'patternfly-react';
+import Tagging from './tagging';
+
+class TaggingWithButtons extends React.Component {
+  onTagCategoryChange = (selectedTagCategory) => {
+    this.props.onTagCategoryChange(selectedTagCategory);
+  }
+
+  onTagValueChange = (selectedTagValue) => {
+    if (this.props.multiValue) {
+      this.props.onTagMultiValueChange({ tagCategory: this.props.selectedTagCategory, tagValue: selectedTagValue });
+    } else {
+      this.props.onTagValueChange({ tagCategory: this.props.selectedTagCategory, tagValue: selectedTagValue });
+    }
+  }
+
+  onTagDeleteClick = (tagCategory, tagValue) => {
+    this.props.onTagDeleteClick({ tagCategory, tagValue });
+  }
+
+  render() {
+    return (
+      <Grid>
+        <Tagging
+          tags={this.props.tags}
+          assignedTags={this.props.assignedTags}
+          onTagValueChange={this.props.onTagValueChange}
+          onTagMultiValueChange={this.props.onTagMultiValueChange}
+          onTagCategoryChange={this.props.onTagCategoryChange}
+          onTagDeleteClick={this.props.onTagDeleteClick}
+          selectedTagCategory={this.props.selectedTagCategory}
+          selectedTagValue={this.props.selectedTagValue}
+        />
+        <Row className={'pull-right'}>
+          <ButtonGroup>
```

Comment:
I would put ButtonGroup in seperate component so we have easier way of combining these two components together.